tests/integration/defs/disaggregated/test_auto_scaling.py

```
@@ -192,17 +239,43 @@ def verify_cluster_info(ready,
+def terminate(*args, show_log_lines=30):
     for arg in args:
-        if arg and isinstance(arg, subprocess.Popen):
+        if arg and isinstance(arg, ProcessWrapper):
             try:
-                arg.kill()
-                arg.wait(timeout=10)
+                # tail the log file for better debugging on CI
+                if os.path.exists(arg.log_path):
+                    print(f"-------------{arg.log_path}---------------")
+                    print(tail(arg.log_path, show_log_lines))
+            except Exception as e:
```

Ruff (0.14.7)

Do not catch blind exception: `Exception`

(BLE001)